tests/README.md

```diff
@@ -0,0 +1,26 @@
+* [Run the osbuilder tests](#run-the-osbuilder-tests)
+* [Further information](#further-information)
+
+## Run the osbuilder tests
+
+osbuilder provides a test script that will create all images and initrds for
```

Comment:
Lines 6-7 suggested rewrite: 
osbuilder provides a test script that creates all images and initrds for all supported distributions and then tests them to ensure a Kata Container can